src/platform/plugins/shared/data/common/search/aggs/agg_configs.ts

```diff
@@ -472,9 +472,8 @@ export class AggConfigs {
     if (!this.hasTimeShifts()) {
       return response;
     }
-    let transformedRawResponse = response.rawResponse;
+    const transformedRawResponse = structuredClone(response.rawResponse);
```

Comment:
I think this was intentionally done inside the `if` statement: https://github.com/elastic/kibana/pull/216519/files#r2024328143